packages/db-mongodb/src/queries/sanitizeQueryValue.ts

```diff
@@ -136,6 +136,14 @@ export const sanitizeQueryValue = ({
       }
     }
 
+    if (operator === 'not_equals' && formattedValue && typeof formattedValue === 'string') {
+      return {
+        rawQuery: {
```

Comment:
I don't see why we need to do this other than we have a bug in buildSearchParam and you're simply dodging it using rawQuery.